### DISCUSSION TITLE
如何在他人复制博客代码时，尾部添加上知识共享协议

### DISCUSSION THREAD

**Discussion by AB-IN-lsy** [liuyib/hexo-theme-stun#264] [Q&A（反馈问题、提问、报告 Bug 等）]
如题，如何在他人复制博客代码时，尾部添加上知识共享协议

比如我的一篇代码，别人复制完后，代码后面我希望跟上作者和url

**Comment by BestJarvan** [comment]
已经新增该功能，等待作者通过PR :)
复制后会在最后一行新增作者信息
```javascript
作者：xxx
地址：xxx
```
https://github.com/liuyib/hexo-theme-stun/pull/308
